components/brave_shields/core/common/features.h

```diff
@@ -13,6 +13,7 @@
 
 namespace brave_shields {
 namespace features {
+BASE_DECLARE_FEATURE(kAdBlockDefaultResourceUpdateInterval);
```

Comment:
all other features are called `kBraveAdblock...`. Don't you want to keep the same style?

Reply:
ideally yes, but it was that way prior to this PR and we probably don't want to touch the naming at this point (it's already immortalized in the Griffin config since https://github.com/brave/brave-variations/pull/783)